### PR TITLE
feat: release 1.4.x

### PR DESCRIPTION
**Pull request by TorstenDittmann**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/models.d.ts [hunk 1]
```diff
@@ -292,6 +331,10 @@ export namespace Models {
          * Database update date in ISO 8601 format.
          */
         $updatedAt: string;
+        /**
+         * If database is enabled. Can be &#039;enabled&#039; or &#039;disabled&#039;. When disabled, the database is inaccessible to users, but remains accessible to Server SDKs using API keys.
```
Comment (loks0n) [inline src/models.d.ts:335]: should \&#039;disabled\&#039; be escape?

many instances of this

File: src/models.d.ts [hunk 2]
```diff
@@ -1746,4 +1995,58 @@ export namespace Models {
...
+        /**
+         * Rule update date in ISO 8601 format.
+         */
+        $updatedAt: string;
+        /**
+         * Domain name.
+         */
+        domain: string;
+        /**
+         * Action definition for the rule. Possible values are &quot;api&quot;, &quot;function&quot;, or &quot;redirect&quot;
```
Comment (loks0n) [inline src/models.d.ts:2032]: \&quot; also

**Review by loks0n — APPROVED** [review]

File: src/services/proxy.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
+import { basename } from "https://deno.land/std@0.122.0/path/mod.ts";
```
Comment (loks0n) [inline src/services/proxy.ts:1]: This service shouldn't be here right?

File: src/services/project.ts [hunk 4]
```diff
@@ -0,0 +1,149 @@
+import { basename } from "https://deno.land/std@0.122.0/path/mod.ts";
```
Comment (loks0n) [inline src/services/project.ts:1]: This service shouldn't be here right?

**Review by loks0n — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**